express-back-end/routes/workouts.js

```diff
@@ -38,25 +38,44 @@ module.exports = db => {
   });
 
 
-  router.put("/workouts/:id", (req, res) => {
-    
-    const {newWorkout} = req.body.createWorkout;
+  router.put("/createWorkout", (req, res) => {
+    console.log('FROM FE', req.body);
+    const newWorkout = req.body.workoutData;
+    const workout_name = newWorkout.workoutName;
```

Comment:
snake_case is Ruby, camelCase is JS.
Although it works, it may create issues later if it becomes a key:value pair in an OBJ

Reply:
Refactored snake case.